packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -385,4 +389,54 @@ export class UnfurlService {
 
         return imageUrl;
     }
+
+    async exportDashboard(
+        dashboardUuid: string,
+        user: SessionUser,
+    ): Promise<NotificationPayloadBase['page']> {
```

Comment:
Let's only return the image url and the name. Is going to make this method cleaner and we don't use the other data in the FE